cartoframes/data/enrichment/enrichment_service.py

```diff
@@ -111,38 +152,77 @@ def __process_filters(filters_dict):
     return filters
 
 
-def __get_tables_and_variables(variables):
+def __process_agg_operators(agg_operators, variables):
+    agg_operators_result = agg_operators.copy()
 
-    if isinstance(variables, pd.Series):
-        variables_id = [variables['id']]
-    elif isinstance(variables, pd.DataFrame):
-        variables_id = variables['id'].tolist()
-    else:
-        raise EnrichmentException('Variable(s) to enrich should be an instance of Series or DataFrame')
+    for variable in variables:
+        variable_name = variable.id.split('.')[-1]
```

Comment:
I'd encapsulate this as a method in the `Variable` class.

Reply:
In fact, we can skip this, we can use `variable.name`, my fault.